tests/functional/tests/security/conftest.py

```diff
@@ -11,33 +11,33 @@
     c_uint16,
     c_int
 )
-from tests.utils import get_random_strings, get_random_ints
+from tests.utils.random import RandomStringGenerator, RandomGenerator, DefaultRanges
 
 import pytest
 
 sys.path.append(os.path.join(os.path.dirname(__file__), os.path.pardir))
 
 
-@pytest.fixture(params=get_random_ints(c_uint16))
+@pytest.fixture(params=RandomGenerator(DefaultRanges.UINT16))
 def c_uint16_randomize(request):
     return request.param
 
 
-@pytest.fixture(params=get_random_ints(c_uint32))
+@pytest.fixture(params=RandomGenerator(DefaultRanges.UINT32))
 def c_uint32_randomize(request):
     return request.param
 
 
-@pytest.fixture(params=get_random_ints(c_uint64))
+@pytest.fixture(params=RandomGenerator(DefaultRanges.UINT64))
 def c_uint64_randomize(request):
     return request.param
 
 
-@pytest.fixture(params=get_random_ints(c_int))
+@pytest.fixture(params=RandomGenerator(DefaultRanges.INT))
```

Comment:
I think it could be params=RandomGenerator(), because DefaultRanges INT by default